corehq/motech/repeaters/expression/repeaters.py

```diff
@@ -156,7 +159,7 @@ def form_class_name(self):
 
     @memoized
     def payload_doc(self, repeat_record):
-        return CommCareCase.objects.get_case(repeat_record.payload_id, repeat_record.domain).to_json()
+        return CommCareCase.objects.get_case(repeat_record.payload_id, repeat_record.domain)
```

Comment:
This was the primary change, return the case object and not the serialized case.